src/app/static/src/tests/components/resetConfirmation.test.ts

```diff
@@ -150,6 +150,41 @@ describe("Reset confirmation modal", () => {
         expect(mockContinueEdit.mock.calls.length).toBe(1);
     });
 
+    it("can render translated version note label", () => {
+
+        const mockContinueEdit = jest.fn();
+        const mockNewVersion = jest.fn();
+        const rendered = mount(ResetConfirmation, {
+            propsData: {
+                continueEditing: mockContinueEdit,
+                cancelEditing: jest.fn()
+            },
+            store: createStore(mockNewVersion, {currentUser: 'test.user@example.com'})
+        });
+
+        const store = rendered.vm.$store
+        const noteLabel = rendered.find("#projectNote label")
+        expectTranslated(noteLabel, "Note (Your reason for saving as a new version)",
+            "Remarque (la raison de l'enregistrement en tant que nouvelle version)", store)
+    });
+
+    it("can set and get note value", async() => {
+
+        const mockContinueEdit = jest.fn();
+        const mockNewVersion = jest.fn();
+        const rendered = mount(ResetConfirmation, {
+            propsData: {
+                continueEditing: mockContinueEdit,
+                cancelEditing: jest.fn()
+            },
+            store: createStore(mockNewVersion, {currentUser: 'test.user@example.com'})
+        });
+
+        await rendered.find("#projectNoteControl").setValue("new version note")
+        const textarea = rendered.find("#projectNoteControl").element as HTMLTextAreaElement;
+        expect(textarea.value).toBe("new version note")
```

Comment:
It's not obvious what this test is for - directly setting the value of the textarea and then checking the value doesn't seem to add anything over the "can render translated version note label" test.